README.md

```diff
@@ -14,16 +14,133 @@ At the moment, it supports transferring data between [file locations](https://gi
 
 This project is maintained by [Astronomer](https://astronomer.io).
 
-## Installation
+## Prerequisites
 
-```
+- Apache Airflow >= 2.4.0.
+
+## Install
+
+The apache-airflow-provider-transfers is available at [PyPI](https://pypi.org/project/apache-airflow-provider-transfers/). Use the standard Python
+[installation tools](https://packaging.python.org/en/latest/tutorials/installing-packages/).
+
+To install a cloud-agnostic version of the apache-airflow-provider-transfers, run:
+
+```shell
 pip install apache-airflow-provider-transfers
 ```
 
+You can also install dependencies for using the `UniversalTransferOperator` with popular cloud providers:
+
+```shell
+pip install apache-airflow-provider-transfers[amazon,google,snowflake]
+```
+
+## Quickstart
```

Comment:
It may be worth giving a head's up that the user can choose how to trigger the DAG to be run in Airflow and that we are illustrating two approaches:
1. Spinning up a local Airflow infrastructure using the open-source Astro CLI and Docker
2. Using vanilla Airflow and Python

Reply:
[Done](https://github.com/astronomer/apache-airflow-provider-transfers/pull/69/commits/88ff234490b9a21102bbc1367ffcec92c614e299)